releasenotes/notes/remove-payload-submodule-58fac4aaf3d4f5da.yaml

```
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    The `ddtrace.payload` submodule has been removed.
```

```suggestion
    The ``ddtrace.payload`` submodule has been removed.
```

Sorry, one more change, since this is RST needs the double ticks